app/components/dashboard/in_progress_collection_row_component.html.erb

```diff
@@ -3,7 +3,7 @@
     <%= show_collection_link %>
   </div>
   <div class="col-3">
-    <%= link_to 'Complete draft', edit_collection_path(collection), class: "mb-2 btn btn-primary" %>
+    <%= link_to 'Complete draft', edit_collection_version_path(collection.head), class: "mb-2 btn btn-primary" %>
```

Comment:
This links to the page that actually has the "Deposit" button (the details part of the from) and not the settings part of the form.  This should reduce user confusion when they click the "Complete draft" button from the dashboard.

Reply:
This PR is really complicated, so can you pull this work out into it's own PR?

Reply:
ok, moved to #1267